backend/tests/conftest.py

```diff
@@ -0,0 +1,89 @@
+import boto3
+import pytest
+import os
+from moto import mock_dynamodb
```

Comment:
Good to see the use of mock! Thank you for adding test cases

Reply:
👍